js/api-augment/definitions/schemas.ts

```diff
@@ -66,7 +66,7 @@ export default {
     SchemasRuntimeApi: [
       {
         methods: {
-          get_schema_by_id: {
+          get_by_schema_id: {
```

Comment:
👋 Over here! 👋 

This is the actual fix!

Reply:
🧑🏽‍🔧